kostyor/rest_api.py

```diff
@@ -307,5 +307,33 @@ def cluster_list():
     return jsonify(clusters)
 
 
+@app.route('/clusters/<cluster_id>/services')
+def service_list(cluster_id):
+    try:
+        db_api.get_cluster(cluster_id)
+    except Exception as ex:
```

Comment:
minor: i think the most popular variable name for exceptions is either `e` or `exc`.